setup.cfg

```diff
@@ -45,6 +45,9 @@ parentdir_prefix = distributed-
 addopts = -v -rsxfE --durations=20
 filterwarnings =
     error:Since distributed.*:PendingDeprecationWarning
+
+    # See https://github.com/dask/distributed/issues/4806
+    error:Port:UserWarning:distributed.node
```

Comment:
This _might_ be a bit too optimistic but let's wait and see. It may definitely interfere if the code base is run suing `pytest-xdist`. We will always get port collisions when running in parallel